doc/nrf/ug_partition_manager.rst

```diff
@@ -82,6 +82,112 @@ span: list
    When creating sub_partitions, this option lists what partitions the sub_partitions should span across.
    This property cannot be used together with the `placement` property.
 
+region: string
+   The memory region (address space) this partition resides in.
+   The placement of partitions is solved separately for each region.
+   The runtime and lifetime of a region is not solved separately.
+   Also, the regions share the same namespace for partition names.
+
+runtime: dict
+   When this partition is active (running).
```

Comment:
Does it make sense to say that a non-persistent memory partition is `running` ? I struggle to separate this from `lifetime`. Maybe provide a description of their differences, and when what is used?

At first sight it looks like runtime is persistent memory while lifetime is non-persistent.

Reply:
Yes `runtime` only makes sense for image partitions. I forgot to mention this in the text.

Reply:
Added a sentence.